### PR TITLE
[Drools 4392] Deploy with Fabric 8 Maven plugin

### PR DESCRIPTION
**Pull request by desmax74**
@jiripetrlik  @mariofusco @danielezonca

### CODE REVIEW

**Comment by desmax74** [comment]
jenkins retest this

File: springboot/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,17 @@
-FROM openjdk:8-jre-alpine
-RUN apk add --no-cache curl
+#https://github.com/fabric8io-images/java/blob/master/images/alpine/openjdk8/jre/Dockerfile
+FROM alpine:3.9
```
[the remark this answers is not in the document]
Comment (desmax74) [reply springboot/Dockerfile]: No, are both from alpine,  but I've found the base images used by fabric 8 and added the commands to choose the jdk

**Comment by desmax74** [comment]
jenkins retest this

**Comment by desmax74** [comment]
jenkins retest this

File: springboot/pom.xml [hunk 2]
```diff
@@ -95,6 +96,27 @@
...
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <version>${version.fabric8.docker-plugin}</version>
+        <configuration>
+          <images>
+            <image>
+              <name>quickstarter/openshift-kie-springboot</name>
```
Comment (jiripetrlik) [inline springboot/pom.xml]: Again. Can we customize?

File: jdkhttp/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,14 @@
-FROM openjdk:8-jre-alpine
-RUN apk add --no-cache curl
+#https://github.com/fabric8io-images/java/blob/master/images/alpine/openjdk8/jre/Dockerfile
+FROM alpine:3.9
```
[the remark this answers is not in the document]
Comment (danielezonca) [reply jdkhttp/Dockerfile]: +1
Considering this doc https://maven.fabric8.io/#generator-java-exec I would suggest to use fabric8/java-jboss-openjdk8-jdk here for community that has an equivalent based on RHEL

File: jdkhttp/README.md [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
+```sh
+oc login -u system:admin
+eval $(minishift oc-env) 
+eval $(minishift docker-env)
+oc project my-kafka-project
+```
+
+#### Relax RBAC for configmap
+```sh
+kubectl create clusterrolebinding permissive-binding --clusterrole=cluster-admin --group=system:serviceaccounts
```
[the remark this answers is not in the document]
Comment (desmax74) [reply jdkhttp/README.md]: I've to test the clusterrole=edit to avoid the cluster-admin

File: springboot/pom.xml (hunk 2 above)
Comment (desmax74) [reply springboot/pom.xml]: ok

File: jdkhttp/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (desmax74) [reply jdkhttp/README.md]: Fixed

**Comment by desmax74** [comment]
jenkins retest this

**Comment by desmax74** [comment]
jenkins retest this

**Comment by desmax74** [comment]
jenkins retest this

**Pull request merged**